Lib/posixpath.py

```diff
@@ -68,26 +68,26 @@ def isabs(s):
 # Ignore the previous parts if a part is absolute.
 # Insert a '/' unless the first part is empty or already ends in '/'.
 
-def join(a, *p):
+def join(path, *paths):
```

Comment:
We don't normally make these sorts of changes unless the function is being modified for another reason. Otherwise the number of PRs making cosmetic improvements will far exceed our ability to review them! Hope that makes sense?

Reply:
That makes sense, and I do agree in general, I was just hoping that the scope of syncing the docstrings and args of the join functions was sufficiently targeted and motivated to justify approval.  However, it was incorrect anyway and I have now reverted the change.  I do still think it would be a worthwhile change to make, but I totally appreciate the point.